homeassistant/components/fritz/sensor.py

```diff
@@ -97,38 +101,38 @@ class SensorData(TypedDict, total=False):
     "kb_s_sent": SensorData(
         name="kB/s sent",
         state_class=STATE_CLASS_MEASUREMENT,
-        unit_of_measurement="kB/s",
+        unit_of_measurement=DATA_RATE_KILOBYTES_PER_SECOND,
         icon="mdi:upload",
         state_provider=_retrieve_kb_s_sent_state,
     ),
     "kb_s_received": SensorData(
         name="kB/s received",
         state_class=STATE_CLASS_MEASUREMENT,
-        unit_of_measurement="kB/s",
+        unit_of_measurement=DATA_RATE_KILOBYTES_PER_SECOND,
         icon="mdi:download",
         state_provider=_retrieve_kb_s_received_state,
     ),
     "max_kb_s_sent": SensorData(
-        name="Max kb/s sent",
-        unit_of_measurement="kb/s",
+        name="Max kB/s sent",
+        unit_of_measurement=DATA_RATE_KILOBYTES_PER_SECOND,
         icon="mdi:upload",
         state_provider=_retrieve_max_kb_s_sent_state,
     ),
     "max_kb_s_received": SensorData(
-        name="Max kb/s received",
-        unit_of_measurement="kb/s",
+        name="Max kB/s received",
+        unit_of_measurement=DATA_RATE_KILOBYTES_PER_SECOND,
```

Comment:
Unit was `kb` not `kB ` ;-)

Simone

Reply:
I saw that, but the unit is still kilobytes per second, is it not?

Reply:
kb = DATA_RATE_KILOBITS_PER_SECOND

kB = DATA_RATE_KILOBYTES_PER_SECOND

Simone

Reply:
Thanks for letting me know! I will update it!
I assumed it was just missing capitalization, as `kilobit` is usually `kbit`.

Reply:
For reference #53274